docs/developer-guide/extensions.md

```diff
@@ -76,7 +76,11 @@ The extension definition will import or define all the needed dependencies (comp
 ### Dynamic import of extension
 
 MapStore supports dynamic import of plugins and extensions.
-In this case, plugin or extension is loaded when app demands to render it for the first time.
+
+Dynamically imported plugin or extension uses lazy-loading: component, reducers and epics will be loaded when extension
```

Comment:
@alexander-fedorenko as I told you in the call, it would be good to clarify somehow that the chunk of a plugin is loaded if you are loading a page that has that plugin in configuration. The current test can confuse the reader that can understand that a plugin is loaded when you activate it from the UI (that's a different thing that require further enhancement).

Reply:
@tdipisa Thanks, updated text to make it clear. I've also added a note explaining how it works if application context have optional plugins available for selection via `User Extensions` library.